pkg/kubernetes/deployment/extnv1beta1/v1alpha1/deployment.go

```diff
@@ -17,9 +17,19 @@ limitations under the License.
 package v1alpha1
 
 import (
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	stringer "github.com/openebs/maya/pkg/apis/stringer/v1alpha1"
 	"github.com/pkg/errors"
+	"k8s.io/api/core/v1"
 	extnv1beta1 "k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// StoragePoolKindCSPC holds the value of CStorPoolCluster
+	StoragePoolKindCSPC = "CStorPoolCluster"
+	// APIVersion holds the value of OpenEBS version
+	APIVersion = "openebs.io/v1alpha1"
```

Comment:
we need to find a better name & better place. Since this version can be used for many resources.
can we have this in `pkg/apis/openebs.io/v1alpha1/`